perf/benchmark/runner/runner.py

```diff
@@ -183,8 +185,9 @@ def run(self, conn, qps, size, duration):
             cacert_arg = "-cacert {cacert_path}".format(cacert_path=self.cacert)
 
         fortio_cmd = (
-            "fortio load -c {conn} -qps {qps} -t {duration}s -a -r {r} {cacert_arg} {grpc} -httpbufferkb=128 " +
+            "fortio load -H={headers} -c {conn} -qps {qps} -t {duration}s -a -r {r} {cacert_arg} {grpc} -httpbufferkb=128 " +
```

Comment:
@howardjohn I think `--header` or `--headers` just the flag that we want to pass values to. For fortio, it actually use `-H` underneath. So, what do you think?